commonwealth-vlr-engine.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.test_files    = s.files.grep(%r{^(test|spec|features)/})
   s.require_paths = ['lib']
 
-  s.required_ruby_version = '~> 2.6'
+  s.required_ruby_version = '~> 2.7'
```

Comment:
Actually don't want to do this since this will break public and 3. you're better off doing `>= 2.6, < 3"